README.md

```diff
@@ -7,19 +7,22 @@ Yet another F-Droid client.
 
 ## Description
 
-Unofficial F-Droid client that resembles the classic F-Droid client.
+Unofficial F-Droid client resembling the classic F-Droid client.
 
-The client supports all basic F-Droid features, such as repository management and package installation. It doesn't
-support privileged extension, root installation, or local repositories.
+Jump over the lazy dog, manage repositories and install software quickly.
+No privileged extension, root installation, or sharing local repositories nearby yet.
 
-### Features
+## Features
 
-* Classic F-Droid style, without cards or inappropriate animations
-* Fast repositories synchronization under good network conditions
-* Built upon standard Android components and a minimal set of dependencies
-* Made with attention to details, both in program logic and design
+* Classic F-Droid style.
+* No cards or inappropriate animations.
+* All apps shown in "Available".
+* Fast repository syncing.
+* Standard Android components and minimal dependencies.
+* Attention to detail in programming logic and design.
+* New-and-shiny, _and_ "old one was better".
```

Comment:
I also think that this point is not necessary.

Reply:
```suggestion
```
Was a joke, will remove. The idea was that nobody can complain, as those are the two qualities people go for, and here they meld.